e_url.php

```diff
@@ -22,9 +22,9 @@ function config($pofile=null)
 		$config = array();
 
 		$config['download'] = array(
-			'regex'			=> '^{alias}\/download/([\d]*)$',
+			'regex'			=> '^{alias}\/dl/([\d]*)$',
 			'redirect'		=> '{e_PLUGIN}vstore/vstore.php?download=$1&',
-			'sef'			=> '{alias}/download/{item_id}'
+			'sef'			=> '{alias}/dl/{item_id}'
```

Comment:
How about 'request' (as used in other parts of e107)  or 'get' instead of 'dl' ?

Reply:
Good idea! I'll change it to 'request'.